testsuite/forge.py

```diff
@@ -994,6 +1001,7 @@ def sanitize_forge_namespace(forge_namespace: str) -> str:
 @envoption("FORGE_NAMESPACE_KEEP")
 @envoption("FORGE_NAMESPACE_REUSE")
 @envoption("FORGE_ENABLE_HAPROXY")
+@envoption("FORGE_ENABLE_FAILPOINTS")
```

Comment:
How does this option actually get triggered?

It seems like the failpoints might depend on the actual test suite being called?

Reply:
For now, the test suite that uses failpoints must be triggered with `FORGE_ENABLE_FAILPOINTS`. I imagine there's a better way to wrap it once the suite is actually written, the same way we have some special logic for the `compat` test suite as well